networking/hardware_networks/about-sriov.adoc

```diff
@@ -5,12 +5,18 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can use Single Root I/O Virtualization (SR-IOV) network devices with additional networks on your {product-title} cluster for high performance applications.
+Developed by the PCI-SIG (PCI Special Interest Group), the Single Root I/O Virtualization (SR-IOV) specification is a standard for a type of PCI device assignment that can share a single device with multiple pods.
 
-[id="about-using-sr-iov-hardware"]
-== About SR-IOV hardware on {product-title}
+SR-IOV enables you to segment a compliant network device, recognized on the host node as a Physical Function (PF), into multiple Virtual Functions (VFs).
+Within a container, the VF is available in the `netns` of the container and the VF can be used like any other network device.
```

Comment:
@zshi-redhat, does it actually work this way, or is the VF available in a pod/container in some other way?

Reply:
Depending on the driver(netdevice, vfio-pci) VF binds to, it appears as a regular kernel network device in container netns or a character device with device spec (e.g. vfio/<iommu_group>) mounted to container.

Reply:
@zshi-redhat, updated!